ert_shared/libres_facade.py

```diff
@@ -11,21 +11,21 @@ class LibresFacade(object):
     """Facade for libres inside ERT."""
 
     def __init__(self, enkf_main):
-        self._enkf_main = enkf_main
+        self.enkf_main = enkf_main
```

Comment:
why the change from indicating private to public?  the whole purpose of the facade is to wrap enkf_main and we don't want anyone to use it directly 🤔

Reply:
That is perhaps correct. The reason is that extraction uses `ERT.ert` and I needed to replace that with something else, but it is probably better to dig deeper and get rid of that line.

Reply:
yes - the ERT.ert thing.... 🙈  But at least it's hidden inside the facade... But as stated in the message below - if you feel the need for this to change - we can discuss ! And if we decide this is the way to do it - then at least i want two commits - one which contains the renaming and one with the actual logic 👍